docs/source/usersguide/materials.rst

```diff
@@ -0,0 +1,169 @@
+:envvar:`OPENMC_CROSS_SECTIONS` environment variable), it will attempt to only
+put isotopes in your model for which you have cross section data. In the case of
+oxygen in ENDF/B-VII.1, the abundance of O18 would end up being lumped with O16.
+
+-----------------------
+Thermal Scattering Data
+-----------------------
+
+If you have a moderating material in your model like water or graphite, you
+should assign thermal scattering data (so-called :math:`S(\alpha,\beta)`) using
+the :meth:`Material.add_s_alpha_beta` method. For example, to model light water,
+you would need to add hydrogen and oxygen to a material and then assign the
+``c_H_in_H2O`` thermal scattering data::
+
+   water = openmc.Material()
+   water.add_nuclide('H1', 2.0)
+   water.add_nuclide('O16', 1.0)
+   water.add_s_alpha_beta('c_H_in_H2O')
+   water.set_density('g/cm3', 1.0)
+
+.. _usersguide_naming:
+
+------------------
+Naming Conventions
+------------------
+
+OpenMC uses the GND_ naming convention for nuclides, metastable states, and
+compounds:
+
+:Nuclides: ``SymA`` where "A" is the mass number (e.g., ``Fe56``)
+:Elements: ``Sym0`` (e.g., ``Fe0`` or ``C0``)
+:Excited states: ``SymA_eN`` (e.g., ``V51_e1`` for the first excited state of
+                 Vanadium-51.) This is only used in decay data.
+:Metastable states: ``SymA_mN`` (e.g., ``Am242_m1`` for the first excited state
+                    of Americium-242).
+:Compounds: ``c_String_Describing_Material`` (e.g., ``c_H_in_H2O``). Used for
+            thermal scattering data.
+
+.. important:: The element syntax, e.g., ``C0``, is only used when the cross
+               section evaluation is an elemental evaluation, like carbon in
+               ENDF/B-VII.1! If you are adding an element via
+               :meth:`Material.add_element`, just use ``Sym``.
+
+.. _GND: https://www.oecd-nea.org/science/wpec/sg38/Meetings/2016_May/tlh4gnd-main.pdf
+
+
+-----------
+Temperature
+-----------
+
+Some Monte Carlo codes define temperature implicitly through the cross section
+data, which is itself given only at a particular temperature. In OpenMC, the
+material definition is decoupled from the specification of temperature. Instead,
+temperatures are assigned to :ref:`cells <usersguide_cells>`
+directly. Alternatively, a default temperature can be assigned to a material
+that is to be applied to any cell where the material is used. In the absence of
+any cell or material temperature specification, a global default temperature can
+be set that is applied to all cells and materials. Anytime a material
+temperature is specified, it will override the global default
+temperature. Similarly, anytime a cell temperatures is specified, it will
+override the material or global default temperatures.
+
+To assign a default material temperature, one should use the ``temperature``
+attribute, e.g.,
+
+::
+
+   hot_fuel = openmc.Material()
+   hot_fuel.temperature = 1200.0  # temperature in Kelvin
```

Comment:
I *think* that the fact that the temperatures are expressed in units of Kelvin is first mentioned in this code comment. Perhaps mention this in the first or second sentence of the paragraph above?